D1371R3.md

```diff
@@ -371,8 +370,8 @@ Op parseOp(Parser& parser) {
 ## Basic Syntax
 
 > | `inspect constexpr`*~opt~* `(` *init-statement~opt~* *condition* `)` *trailing-return-type~opt~* `{`
-> |     *pattern* *guard~opt~* `=>` *expression* `,`
-> |     *pattern* *guard~opt~* `:` *statement*
+> |     *pattern* *guard~opt~* `:` *expression* `,`
```

Comment:
This should be `statement` without the comma.

Reply:
Done